test/extended/util/annotate/rules.go

```diff
@@ -68,7 +68,12 @@ var (
 			`\[sig-network-edge\]\[Feature:Idling\] Unidling should work with TCP \(while idling\)`,
 		},
 		// tests that may work, but we don't support them
-		"[Disabled:Unsupported]": {},
+		"[Disabled:Unsupported]": {
+			// Skip vSphere-specific storage tests. The standard in-tree storage tests for vSphere
+			// (prefixed with `In-tree Volumes [Driver: vsphere]`) are enough for testing this plugin.
+			// https://bugzilla.redhat.com/show_bug.cgi?id=2019115
+			`\[sig-storage\].*\[Feature:vsphere\]`,
```

Comment:
This doesn't seem to match to me.  I see a lot of `[sig-storage] In-tree Volumes [Driver: vsphere] [Testpattern:****` failures.

Reply:
Actually, it's more targeted than that.  Some of those do work, but some fail consistently.

Reply:
@deads2k the regex above matches `Feature:vsphere`, not `Driver:vsphere`. We don't want to skip the latter.

Reply:
@bertinatto there are currently 13 `Driver:vsphere` tests that are broken due to:
~~~
Failed to bootstrap vSphere with error: env variable 'VSPHERE_CONF_FILE' is not set
~~~

It looks like we are intending to skip these [tests](https://github.com/openshift/origin/pull/26565/files#diff-998be43366fe821c61ca242aa34949870c9c6df2572cc060000e4cd990a72bebR80) but those test still appear to be getting run.

Reply:
@rvanderp3 basically we are skipping two set of vSphere tests:
1. All `Feature:vsphere` tests.
2. [Only 4](https://github.com/openshift/origin/pull/26565/files#diff-998be43366fe821c61ca242aa34949870c9c6df2572cc060000e4cd990a72bebR77-R80) `Driver:vsphere` tests that require topology, which we don't support in our CI.

All the remaining `Driver:vsphere` tests should not be skipped and I'm trying to fix the failing ones [here](https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openshift_release/22993/rehearse-22993-pull-ci-openshift-vmware-vsphere-csi-driver-master-e2e-vsphere/1456694637673582592).

Here's a rehearsal job that passed: https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openshift_release/22993/rehearse-22993-pull-ci-openshift-vmware-vsphere-csi-driver-master-e2e-vsphere/1456694637673582592

Reply:
> @rvanderp3 basically we are skipping two set of vSphere tests:
> 
> 1. All `Feature:vsphere` tests.
> 2. [Only 4](https://github.com/openshift/origin/pull/26565/files#diff-998be43366fe821c61ca242aa34949870c9c6df2572cc060000e4cd990a72bebR77-R80) `Driver:vsphere` tests that require topology, which we don't support in our CI.
> 
> All the remaining `Driver:vsphere` tests should not be skipped and I'm trying to fix the failing ones [here](https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openshift_release/22993/rehearse-22993-pull-ci-openshift-vmware-vsphere-csi-driver-master-e2e-vsphere/1456694637673582592).
> 
> Here's a rehearsal job for that PR: https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openshift_release/22993/rehearse-22993-pull-ci-openshift-vmware-vsphere-csi-driver-master-e2e-vsphere/1456694637673582592

@bertinatto ok, I started a [PR](https://github.com/openshift/origin/pull/26575) to skip those which may not be necessary at this point.  Sorry for the noise!  I'll close that out.